packages/ckeditor5-mention/src/mentionui.js

```diff
@@ -220,6 +220,13 @@ export default class MentionUI extends Plugin {
 		mentionsView.items.bindTo( this._items ).using( data => {
 			const { item, marker } = data;
 
+			// See: #10479
```

Comment:
Such note without any context doesn't bring much more than checking git blame. The idea usually is to give a short, one or two sentences long explanation and then, if there is more to read on GH, leave a ticket reference. 
Here I think the most important part is that the default `10` is kept for backwards compatibility. Sure, we can still leave the ticket reference after that, but it wouldn't be a tragedy if we didn't as there is not much information left.